libbeat/template/load.go

```diff
@@ -80,6 +82,41 @@ func (l *Loader) Load() error {
 	return nil
 }
 
+func (l *Loader) Generate() error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Loader.Generate should have comment or be unexported